src/components/Vault/BasicVault/VaiCard.tsx

```diff
@@ -95,68 +90,43 @@ function VaultCard({ settings }: VaultCardProps) {
 
   return (
     <VaultCardWrapper>
-      <div className={`header-container ${expanded ? '' : 'fold'}`}>
-        <Row className="header">
-          <Col className="col-item" lg={{ span: 3 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">Stake</div>
-            <div className="content">
-              <img src={vaiImg} alt="Vai" />
-              <span>VAI</span>
-            </div>
-          </Col>
-          <Col className="col-item" lg={{ span: 3 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">Earn</div>
-            <div className="content">
-              <img src={xvsImg} alt="rewardToken" />
-              <span>XVS</span>
-            </div>
-          </Col>
-          <Col className="col-item" lg={{ span: 4 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">Reward Pool</div>
-            <div className="content">
-              {commaFormatter(
-                totalPendingRewards
-                  .div(1e18)
-                  .dp(4, 1)
-                  .toString(10),
-              )}{' '}
-              XVS
-            </div>
-          </Col>
-          <Col className="col-item" lg={{ span: 4 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">VAI Staking APR</div>
-            <div className="content">{settings.vaiAPY}%</div>
-          </Col>
-          <Col className="col-item" lg={{ span: 4 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">Total VAI Staked</div>
-            <div className="content">
-              {settings.vaultVaiStaked
-                ? commaFormatter(new BigNumber(settings.vaultVaiStaked).dp(4, 1).toString(10))
-                : 0}{' '}
-              VAI
-            </div>
-          </Col>
-          <Col className="col-item" lg={{ span: 4 }} md={{ span: 6 }} xs={{ span: 12 }}>
-            <div className="title">XVS Daily Emission</div>
-            <div className="content">{commaFormatter(dailyEmission.toString(10))} XVS</div>
-          </Col>
-          <Col
-            className="col-item expand-icon-wrapper"
-            lg={{ span: 2 }}
-            xs={{ span: 24 }}
-            onClick={() => setExpanded(!expanded)}
-          >
-            <img className="expand-icon" alt="open" src={arrowDownImg} />
-          </Col>
-        </Row>
-      </div>
+      <CardHeader
+        stakedToken="VAI"
+        rewardToken="XVS"
+        apy={settings.vaiAPY || 0}
+        totalStakedAmount={new BigNumber(settings.vaultVaiStaked || 0)}
+        totalPendingRewards={totalPendingRewards.div(1e18)}
+        dailyEmission={dailyEmission}
+        onExpand={() => {
+          setExpanded(!expanded);
+        }}
+      />
       <div className="content-container">
         {expanded ? (
```

Comment:
```suggestion
        {expanded && (
```
Then see my next comment line 131...

Reply:
got it